build/nm_big.json

```diff
@@ -19802,14 +19802,13 @@
         "q=access_level:open&fq=res_format:(GeoJSON OR \"OGC WMS\" OR \"OGC WFS\" OR WFS OR \"API ArcGIS Server Feature Service\" OR \"API ArcGIS Server Map Service\" OR \"API ArcGIS Map Service\" OR \"Esri REST\")"
       ],
       "includeCsv": true,
-      "includeGeoJson": true,
+      "includeGeoJson": false,
       "includeKml": true,
       "includeWms": true,
-      "includeWfs": true,
+      "includeWfs": false,
       "includeEsriMapServer": true,
       "includeEsriFeatureServer": true,
```

Comment:
You should exclude ESRI Feature Server too.

Reply:
done.